modules/nlu/src/views/full/index.tsx

```diff
@@ -31,7 +31,12 @@ const NLU: FC<Props> = props => {
   const [intents, setIntents] = useState([])
   const [entities, setEntities] = useState([])
 
-  const loadIntents = () => api.fetchIntents().then(setIntents)
+  const loadIntents = () =>
+    api
+      .fetchIntents()
+      .then(setIntents)
+      .then(x => setCurrentItem(undefined))
+      .then(x => setCurrentItem(currentItem)) // this is little hack to trigger update for IntentEditor->Slots->SlotModal
```

Comment:
This "little hack" will make the UI glitch on the SplashScreen

Reply:
May be you have better idea, how to refresh modal window without of this glitch?

Reply:
I haven't thought of it but I'm sure there's something we can do. We can address that later though